### PR TITLE
Fix #275: Moves variables from env to with

### LINKED ISSUE #275: [Github action] CLA Check - Input variables using `with` instead of `env`.
**Describe the bug**
The CLA Check GitHub action has to take the secrets as input via `with` (like how it is done for GITHUB_TOKEN) instead of `env` (Like how it is currently done).
Change all similar code https://github.com/oppia/oppiabot/blob/bad457439a6c65cbe3d7b93ef81a77b041d291bf/actions/src/pull_requests/claCheck.js#L29-L30 to https://github.com/oppia/oppiabot/blob/bad457439a6c65cbe3d7b93ef81a77b041d291bf/actions/src/pull_requests/claCheck.js#L46

This was requested by @BenHenning https://github.com/oppia/oppia-android/pull/3166#discussion_r646012454

### PR DESCRIPTION
**Pull request by gp201**
## Explanation

This PR fixes #275 
This PR changes Input variables using `env` to use `with` instead

## Checklist
- [x] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [x] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

**Comment by gp201** [comment]
@vojtechjelinek @jameesjohn PTAL

File: action.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Oppiabot Actions
+name: Praneeth Actions
```
Comment (vojtechjelinek) [inline action.yml:1]: Remove this change :smile:

File: actions/src/dispatcher.js [hunk 2]
```diff
@@ -31,8 +31,10 @@ module.exports = {
     const repoName = context.payload.repository.name.toLowerCase();
     const checksWhitelist = constants.getChecksWhitelist();
     if (Object.prototype.hasOwnProperty.call(checksWhitelist, repoName)) {
+      core.info('ran1');
```
Comment (vojtechjelinek) [inline actions/src/dispatcher.js:34]: Why is this needed?

File: constants.js [hunk 3]
```diff
@@ -133,6 +133,10 @@ const checksWhitelist = {
     [pullRequestReviewEvent]: [pullRequestReviewCheck],
     [issueCommentCreatedEvent]: [respondToReviewCheck],
   },
+  'test-github-actions': {
+    [openEventGithubActions]: [claCheckGithubAction],
+    [reopenEventGithubActions]: [claCheckGithubAction]
+  }
```
Comment (vojtechjelinek) [inline constants.js:139]: Should this be removed?

File: spec/claCheckSpec.js [hunk 4]
```diff
@@ -31,9 +31,15 @@ const sheetsCredentials =
...
-    process.env.SHEETS_TOKEN = sheetsToken;
-    spyOn(core, 'getInput').and.returnValue('sample-token');
+    spyOn(core, 'getInput').and.callFake((val) =>{
+      if (val === 'repo-token') {
+        return 'sample-token';
+      } else if (val === 'sheets-cred') {
+        return sheetsCredentials;
+      } else if (val === 'sheets-token') {
+        return sheetsToken;
+      }
```
Comment (vojtechjelinek) [inline spec/claCheckSpec.js:41]: Should this function be separated so that we get rid of the duplicity?

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thanks! Left a few comments.

File: action.yml (hunk 1 above)
Comment (gp201) [reply action.yml]: Done

File: actions/src/dispatcher.js (hunk 2 above)
Comment (gp201) [reply actions/src/dispatcher.js]: Removed

File: constants.js (hunk 3 above)
Comment (gp201) [reply constants.js]: Removed

File: spec/claCheckSpec.js (hunk 4 above)
Comment (gp201) [reply spec/claCheckSpec.js:41]: No, Since the function is required to return different values for one of the tests.

**Review by vojtechjelinek — APPROVED** [review]
LGTM!

**Review by jameesjohn — APPROVED** [review]
LGTM
